setup.py

```diff
@@ -15,9 +15,9 @@
 
 setup(
     name='seqseek',
-    version='0.3.3',
+    version='0.3.4',
```

Comment:
should this be 0.4.1?

Reply:
yeah we should bump the minor version; this isn't a patch.

Reply:
Version in setup.py now set to 0.4.1. However, does not appear to work with Python 3.6. Tried it out by adding py36 to tox.ini envlist and running make test. Just mentioning it since I believe we said 0.4.0 should have been the version that added python 3 compatibility.

Reply:
Seems to be passing in CI: https://travis-ci.org/23andMe/seqseek/builds/523623762?utm_source=github_status&utm_medium=notification

Looks like your fork might be behind. Changes look good though. Merging.